batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/util/PasswordGenerator.java

```diff
@@ -0,0 +1,21 @@
+package com.lftechnology.batch7crud.util;
+
+import java.math.BigInteger;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.logging.Logger;
+
+/**
+ * Created by Grishma Shrestha grishmashrestha@lftechnology.com on 1/29/16.
+ */
+public class PasswordGenerator {
+  private static final Logger LOGGER = Logger.getLogger(PasswordGenerator.class.getName());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOGGER" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)